test/tests/functional/pbs_init_script.py

```diff
@@ -71,6 +71,22 @@ def test_env_vars_precede_pbs_conf_file(self):
         self.assertNotIn("PBS comm", output)
         self.assertIn("PBS mom", output)
 
+    def test_init_services_pid(self):
+        """
+        Test if the pid of PBS daemons are updated correctly
+        after a restart via PBSInitServices's functions
+        """
+        self.server.pi.stop()
+        self.server.pi.start_server()
+        self.server.pi.start_mom()
+        self.server.pi.start_comm()
+        self.server.pi.start_sched()
+
+        self.assertTrue(self.server.signal('-HUP'))
+        self.assertTrue(self.mom.signal('-HUP'))
+        self.assertTrue(self.scheduler.signal('-HUP'))
+        self.assertTrue(self.comm.signal('-HUP'))
+
```

Comment:
Now that you added these this to selftests, you should remove it from here.

Reply:
Ah ! yes. Thanks for pointing that out. Removed it now.